CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 # Git LFS Changelog
 
-## 2.13.2 (13 Jan 2020)
+## 2.13.3 (26 Mar 2021)
+
+This release fixes a bug that caused `git lfs prune` to hang, updates some
```

Comment:
If we add #4449 to this release we should tweak this to note that it fixes two bugs of this type (sigh).